model/change_point_detection.go

```diff
@@ -428,3 +461,18 @@ func createChangePoint(ctx context.Context, env cedar.Environment, resultToUpdat
 	_, err := env.GetDB().Collection(perfResultCollection).UpdateOne(ctx, filter, update)
 	return errors.Wrap(err, "Unable to create change point")
 }
+
+func TriageChangePoint(ctx context.Context, env cedar.Environment, perfResultId string, measurement string, status TriageStatus) error {
```

Comment:
I'd be consistent about using ID rather than Id

Reply:
Sorry, force of habit!